drivers/rtc/rtc_ll_stm32.c

```diff
@@ -442,13 +437,13 @@ static int rtc_stm32_set_time(const struct device *dev, const struct rtc_time *t
 		LL_RTC_DATE_SetWeekDay(RTC, timeptr->tm_wday);
 	}
 
-
 	LL_RTC_TIME_SetHour(RTC, bin2bcd(timeptr->tm_hour));
 	LL_RTC_TIME_SetMinute(RTC, bin2bcd(timeptr->tm_min));
 	LL_RTC_TIME_SetSecond(RTC, bin2bcd(timeptr->tm_sec));
 
 	LL_RTC_DisableInitMode(RTC);
 
+end:
```

Comment:
`goto` name should describe what it does, something like `protect_unlock_return:`